modules/controller_utils/__init__.py

```diff
@@ -168,17 +170,20 @@ def get_filename_safe_identifier() -> str:
         return now.isoformat().replace(':', '_')
 
 
-def get_zone_robot_file_path(zone_id: int) -> Path:
+def get_zone_robot_file_path(zone_id: int, robot_type: RobotTypes) -> Path:
     """
     Return the path to the robot.py for the given zone, without checking for
     existence.
     """
-    return ARENA_ROOT / "zone-{}".format(zone_id) / "robot.py"
+    if robot_type == RobotTypes.CRANE:
```

Comment:
Nit: Could probably just do this with `return ARENA_ROOT / "zone-{}".format(zone_id) / "{}.py".format(robot_type.value)`

Reply:
Fixed in b1b0844b88e8a1103d4180372d9c5d49ee023c32